src/schemas/project.graphql

```diff
@@ -178,6 +186,10 @@ extend type Mutation {
     """
     address: AddressInput!
     """
+    Name of the client
+    """
+    clientName: String!
```

Comment:
Let's try to keep keys at consistent place everywhere. This will be helpful later.

For e.g. here clientName is after address. But in updateProject, it is before address.